src/components/Modal/index.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react'
+// import { Loader } from '../Loader/index';
 
```

Comment:
I'm sure I'll figure this out soon enough, but is there a reason it's commented out? Was react being cranky about something again?

Reply:
Hey Kelly, thanks for asking! I commented it out because we hadn't implemented it yet with logic inside of the Modal component is all and yes, I was afraid React would be cranky about that!